Makefile

```diff
@@ -51,15 +50,14 @@ test: ## Run tests
 # build packages
 
 build: ## Build a release (sdist and wheel)
-	$(python) -m pip install --upgrade setuptools wheel
-	$(python) setup.py sdist bdist_wheel
+	$(python) -m pip install --upgrade build
```

Comment:
Do you think maybe "build" should be in `requirements-dev.txt` (which is installed during `make bootstrap` - see the release process in `README-hacking.md`) instead of being installed here?

Reply:
I think not since it's unrelated to testing; I think `twine` should also not be in there

Reply:
`requirements-dev.txt` is not just for testing - we install these packages when running `make bootstrap` before running `make build`/`make release` as part of the build/release process.

OTOH now I see that we also run `pip install` for `setuptools` and `wheel` as part of `make build`...

Anyway, this is not blocking a merge.